### DISCUSSION TITLE
Grant repo access to an entire organization?

### DISCUSSION THREAD

**Discussion by pwschultz2** [community/community#57885] [Enterprise] [answered]
### Select Topic Area

Question

### Body

I'm part of an organization, but I also have repos in my own user. Is it possible to grant access to everyone in my organization to view my repository under `pwschultz2`? I'll elaborate below:

We have an Enterprise with several Organizations, one of which is `core-developers`<sup>1</sup>. This organization contains multiple repositories and access is managed by `core-developers` contributor memberships.

I have my own repositories under `pwschultz2`, including `pwschultz2/my-shiny-repo`<sup>1</sup>. I want everyone in `core-developers` to be able to see this repository. Is there a way to automate this so that we don't have to manage multiple lists of contributors?

I thought about creating a Team `@core-developers/developers`, but it doesn't seem that I can add that team as a contributor.

<sup>1 For the sake of argument.</sup>

**Comment by aahouzi** [comment, marked as answer]
Yes, it is possible to grant access to everyone in your organization to view your repository under your personal user account (pwschultz2). One way to achieve this is by leveraging teams and their access permissions within your organization.

Here's how you can automate access for everyone in the core-developers organization to view your repository:

1. In your organization (core-developers1), create a team specifically for granting access to your repository. For example, you can create a team called "Viewers."

2. Navigate to your repository (pwschultz2/my-shiny-repo1) and go to the "Settings" tab.

3. Under the "Manage access" section, click on "Invite a team."

4. Select the "Viewers" team that you created in Step 1.

5. Choose the appropriate access level for the team members. For read access, you can assign the "Read" permission.

6. Save the changes, and now everyone in the "Viewers" team within the core-developers organization will have read access to your repository.

By creating a dedicated team within your organization, you can manage access to your repository more efficiently. Adding new members to the "Viewers" team will automatically grant them access to your repository without needing to individually manage contributor memberships.

Please note that if you want to allow organization-wide read access to your repository, you can also consider adjusting the default organization-wide permission settings. However, this may grant access to all repositories within the organization, so use this option with caution and evaluate your organization's specific requirements.

Additionally, ensure that your organization's policies and guidelines align with the access permissions you grant to maintain security and compliance.

**Reply by pwschultz2** [reply]
I was missing the detail in Step 3. Thank you. And yes, the Team I'm creating will be specifically for the organizational group referring to my department, and my relevant repositories are projects for this team.
